account_invoice_fiscal_position_update/models/account_invoice.py

```diff
@@ -47,7 +47,15 @@ def fiscal_position_change(self):
                 if fp:
                     taxes = fp.map_tax(taxes)
 
+                # if we are replacing or adding a tax with "price_include" we
+                # should recompute price_subtotal
+                recompute_subtotal = any(
+                    x.price_include for x in (taxes + line.tax_ids)
+                )
+
                 line.tax_ids = [(6, 0, taxes.ids)]
+                if recompute_subtotal:
```

Comment:
But why we need to check if there are price_include taxes only?
if you will return to taxes out, there will be the same error
so you could remove recumpute subtotal check and live _onchange_price_subtotal call only